src/proof/PGInterAux.cc

```diff
@@ -77,19 +75,16 @@ icolor_t ProofGraph::getPivotColor(ProofNode & n) {
 	interpolationInfo.updateColoringfromAnts(n);
 
     // Determine if variable A-local, B-local or AB-common
-	icolor_t var_color = var_class == icolor_t::I_A || var_class == icolor_t::I_B ? var_class : icolor_t::I_UNDEF;
-	if (var_color == icolor_t::I_UNDEF) {
+	icolor_t var_color = var_class;
+	if (var_color != icolor_t::I_A and var_color != icolor_t::I_B) {
         assert(var_class == icolor_t::I_AB);
         var_color = getSharedVarColorInNode(v, n);
```

Comment:
Assigning to `var_color` seems to be unnecessary at line 78.  Maybe we can declare it directly at line 81?